.circleci/config.yml

```diff
@@ -131,4 +410,4 @@ commands:
             set -xe
             pip3 install --user astronomer_e2e_test
             # We can remove --debug after we are happy with it
-            astronomer-ci --debug publish-github-release --github-repository $CIRCLE_PROJECT_REPONAME --github-organization $CIRCLE_PROJECT_USERNAME --commitish $CIRCLE_SHA1
+            astronomer-ci --debug publish-github-release --github-repository $CIRCLE_PROJECT_REPONAME --github-organization $CIRCLE_PROJECT_USERNAME --commitish $CIRCLE_SHA1
```

Comment:
@sjmiller609 looks like we need a newline here.

Reply:
this is a jinja2 template output, I can modify the script to add a newline at the end?